src/components/QueryBuilder.vue

```diff
@@ -55,12 +56,14 @@ export default ( Vue as VueConstructor<Vue & { $refs: { condition: InstanceType<
 		},
 		validate(): void {
 			// TODO: Clean up FormValue <-> Query relation
-			const formValues = {
-				property: this.$store.getters.query.condition.propertyId,
-				value: this.$store.getters.query.condition.value,
-				propertyValueRelation: this.$store.getters.query.condition.propertyValueRelation,
-			};
-			const validator = new Validator( formValues );
+			const formValues = this.$store.getters.query.conditions.map( ( condition: Condition ) => {
+				return {
+					property: condition.propertyId,
+					value: condition.value,
+					propertyValueRelation: condition.propertyValueRelation,
+				};
+			} );
+			const validator = new Validator( formValues[ 0 ] );
```

Comment:
Ok for now, but we should make sure that we rework this as soon as we actually have multiple queries.